modules/apps/asset/asset-list-web/src/main/resources/META-INF/resources/asset_list/source.jsp

```diff
@@ -583,9 +585,7 @@ List<AssetRendererFactory<?>> classTypesAssetRendererFactories = new ArrayList<>
 		'<portlet:namespace />ddmStructureDisplayFieldValue'
 	);
 
-	dom.delegate(sourcePanel, 'click', '.asset-subtypefields-popup', function (
-		event
-	) {
+	delegate(sourcePanel, 'click', '.asset-subtypefields-popup', function (event) {
```

Comment:
These uncleaned delegate events are dangerous. Being tied to `sourcePanel` means they probably don't matter after you navigate since that node gets destroyed.

That would not be the case if the target was `document` or `body`, so keep an eye out for such issues.

Reply:
That makes sense. I was trying to have minimal changes so I didn't even consider that, I'll do my best to be more careful with that in the future.

Reply:
Nothing to do here, not to worry!

I was just pointing out some known nuisances of how delegate works in the context of DXP so you could catch some of those issues in the future.